frontend/scss/photoView.scss

```diff
@@ -94,12 +94,88 @@ rect {
     padding-left: 10px;
 }
 
+.side-button {
+    font-size: 18px;
+    font-weight: 500;
+    border: 2px solid;
+    padding: 2px 10px;
+    background-color: white;
+}
+
+.side-button:hover {
+    background-color: #20CCD7;
```

Comment:
In a future PR, let's define our hex values inside of a variables file, like here: https://github.com/dhmit/self-sustaining-cities/blob/main/frontend/scss/variables.scss
Then we can use these variables everywhere. The reason for this is if we have to tweak any values, we would just have to change them once in that file as opposed to everywhere they're mentioned